src/main/java/frc/robot/Constants.java

```diff
@@ -107,6 +107,12 @@
 
     //#endregion
 
+    //#region Climber Details
+
+    public static final double CLIMEBR_ENCODER_INCHES_PER_TICK = 1;
```

Comment:
Typo, `CLIMEBR` -> `CLIMBER`

Please also leave a `TODO` comment here so we can update the value later